CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `const` fns for `PublicKey` and `StealthAddress` [#63]
```

Comment:
Add the name of the functions you've added to make it clear.
I would suggest to have one ` - Add` for types

Reply:
Changelog updated